File: 2 Factory/src/ORACLE.java
public class ORACLE implements IDB {
    @Override
    public void selectData() {
        System.out.println("ORACLE veritabanından veri okundu.");
    }
    
    @Override
    public void insertData() {
        System.out.println("ORACLE veritabanına veri yazıldı.");
    }

}

File: 2 Factory/chsarp/ORACLE.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Factory_csharp
{
    public class ORACLE : IDB
    {
        public void SelectData()
        {
            Console.WriteLine("ORACLE veritabanından veri okundu.");
        }

        public void InsertData()
        {
            Console.WriteLine("ORACLE veritabanına veri yazıldı.");
        }
    }
}
